dropbox/async/types.go

```diff
@@ -49,6 +49,10 @@ const (
 func (u *LaunchResultBase) UnmarshalJSON(body []byte) error {
 	type wrap struct {
 		dropbox.Tagged
+		// AsyncJobId : This response indicates that the processing is
+		// asynchronous. The string is an id that can be used to obtain the
+		// status of the asynchronous job.
+		AsyncJobId json.RawMessage `json:"async_job_id,omitempty"`
```

Comment:
Why make this a `json.RawMessage` and then unmarshal it again in line 65?
Could this be a `string` and then  at that point:

- Either just set `u.AsyncJobId = w.AsyncJobId=,
or
- Get rid of `wrap` altogether and `Unmarshal` directly into `u`?

I'm probably missing something.